books/ownCloud_Admin_Manual.adoc

```diff
@@ -10,42 +10,71 @@ The ownCloud Team <docs@owncloud.com>
 :icon-set: octicon
 
 include::modules/admin_manual/pages/index.adoc[leveloffset=+1]
+
```

Comment:
Why is that needed? That doesn't make sense...

Reply:
It did, at least previously, as I commented in the PR that removed the newlines. mojavelinux gave a recommendation to do that when working with asciidoctor-pdf. I don't have a link to further information, but not having them caused corrupted build artifacts previously.

Reply:
What does corrupted artifacts mean? So far the PDFs are building fine, it's just printing warnings while building it.

Reply:
I'd have to dig into what was happening previously to be more specific. If it's working, then I'll update the PR.